packages/compass-e2e-tests/helpers/selectors.ts

```diff
@@ -1055,31 +1055,31 @@ export const queryBar = (tabName: string): string => {
 };
 export const queryBarOptionInputFilter = (tabName: string): string => {
   const tabSelector = collectionContent(tabName);
-  return `${tabSelector} [data-testid="query-bar-option-filter"]`;
+  return `${tabSelector} [data-testid="query-bar-option-filter-input"]`;
```

Comment:
At some point these selectors started pointing to the div elements that wrap the inputs and not the inputs themselves and in previous webdriver versions that somehow kept working with setValue(). Now something in the webdriverio -> webdriver -> chromedriver chain is stricter about it and complains that the element is not interactive. Which makes sense, I guess.